tidb-cloud/serverless-private-link-connection-to-alicloud-rds.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Connect to Alibaba Cloud ApsaraDB RDS for MySQL via a Private Link Connection
+summary: Learn how to connect to an Alibaba Cloud ApsaraDB RDS for MySQL instance using an Alibaba Cloud Endpoint Service private link connection.
+---
+
+# Connect to Alibaba Cloud ApsaraDB RDS for MySQL via a Private Link Connection 
+
+This document describes how to connect to an Alibaba Cloud ApsaraDB RDS for MySQL instance using an Alibaba Cloud Endpoint Service private link connection.
+
+> **Note:**
+>
+> The Private Link Connections for Dataflow feature is in beta. It might be changed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+## Prerequisites
+
+- ApsaraDB RDS for MySQL: ensure you have an existing ApsaraDB RDS for MySQL instance or the permissions required to create one.
+
+- Alibaba Cloud permissions: verify that your account has the following authorizations to manage networking components:
+
+    - Manage load balancer
+    - Manage endpoint services
+
+- {{{ .essential }}} information: confirm that your {{{ .essential }}} is active in Alibaba Cloud. Retrieve and save the following details for later use:
+
+    - Account ID
+    - Availability Zones (AZ)
+
+To view the the Alibaba Cloud account ID and available zones, do the following:
+
+1. In the [TiDB Cloud console](https://tidbcloud.com), navigate to the cluster overview page of the TiDB cluster, and then click **Settings** > **Networking** in the left navigation pane.
+2. On the **Private Link Connection For Dataflow**, click **Create Private Link Connection**.
+3. You can find the Alibaba Cloud account ID and available zones.
+
+## Step 1. Set up an ApsaraDB RDS for MySQL instance
+
+Identify an Alibaba Cloud ApsaraDB RDS for MySQL you want to use, or [set up a new RDS](https://www.alibabacloud.com/help/en/rds/apsaradb-rds-for-mysql/step-1-create-an-apsaradb-rds-for-mysql-instance-and-configure-databases).
+
+To ensure successful connectivity, your ApsaraDB RDS for MySQL instance must meet the following requirements:
+
+- Region match: the instance must reside in the same Alibaba Cloud region as your {{{ .essential }}} cluster.
+- AZ (Availability Zone) availability: the availability zones must overlap with those of your {{{ .essential }}} cluster.
+- Network accessibility: the instance must be accessible within the VPC, with an appropriately configured IP allowlist.
```

Comment:
use whitelist rather than allowlist
<img width="2990" height="718" alt="image" src="https://github.com/user-attachments/assets/07486321-7078-4350-8883-eccfdd7bc243" />